### PR TITLE
AT_09.02.008 | <My Views> Add description

### PR DESCRIPTION
**Pull request by Kseniia-Vlasova**
https://trello.com/c/j4Xlxlp9/1123-at0902008-my-viewsadd-description

### CODE REVIEW

**Comment by matvlesia** [comment]
I don't think you can change the file _package.json_

**Comment by Kseniia-Vlasova** [comment]
> 
Changes in these folders occurred after entering the command "npm install cypress-real-events". Do I need to cancel them?

**Comment by Rom-T** [comment]
> > 
> 
> Changes in these folders occurred after entering the command "npm install cypress-real-events". Do I need to cancel them?

Why did you enter this command? At the lecture, Maria already added this plugin to the project. The main rule: We do not install anything in the project ourselves!

File: cypress/e2e/addViewsDescription.cy.js [hunk 1]
```diff
@@ -8,4 +8,13 @@ describe('My Views | Add Description', () => {
         cy.get('button[name="Submit"]').click()
         cy.get('#description').should('contain', '123')
     });
-});
+
+
+    it('AT_09.02.008_Add description to "My Views"', () => {
+        cy.get('a[href="/me/my-views"]').click()
+        cy.get('#description-link').should('be.visible').click()
+        cy.get('.jenkins-input').should('be.visible').type('My text')
```
Comment (Rom-T) [inline cypress/e2e/addViewsDescription.cy.js:16]: Please move `My text` in the fixtures

File: cypress/e2e/addViewsDescription.cy.js [hunk 2]
```diff
@@ -8,4 +8,13 @@ describe('My Views | Add Description', () => {
...
+        cy.get('button[name=Submit]').click()
+        cy.get('#description > div:nth-child(1)').contains('My text')
```
Comment (Rom-T) [inline cypress/e2e/addViewsDescription.cy.js:18]: Please move `My text` in the fixtures and contains from does not assert. If you want to assert then use the should
`cy.get('#description > div:nth-child(1)').should('have.text', 'My text')`

**Comment by Kseniia-Vlasova** [comment]
> > > 
> > 
> > 
> > Changes in these folders occurred after entering the command "npm install cypress-real-events". Do I need to cancel them?
> 
> Why did you enter this command? At the lecture, Maria already added this plugin to the project. The main rule: We do not install anything in the project ourselves!

Maybe I didn’t understand something, but these steps were also spelled out in the presentation.
I will try to fix it

File: cypress/e2e/addViewsDescription.cy.js (hunk 1 above)
Comment (Kseniia-Vlasova) [reply cypress/e2e/addViewsDescription.cy.js:16]: I moved 'My text' in fixtures and it works with '.contains' method. 
Is it ok?

**Comment by Rom-T** [comment]
ou have deleted your test. Write to me on Slack

**Review by Rom-T — APPROVED** [review]

**Review by vlada4nik — APPROVED** [review]

**Pull request merged**